babel/numbers.py

```diff
@@ -1010,6 +1011,8 @@ def parse_number(
     1099
     >>> parse_number('1.099', locale='de_DE')
     1099
+    >>> parse_number('1 099', locale='ru')
```

Comment:
If the intent is to test a string that has a non-breaking space character, make it obvious:
```suggestion
    >>> parse_number('1\xa0099', locale='ru')
```